core/src/test/scala-3/cats/derived/SemigroupKSuite.scala

```diff
@@ -0,0 +1,61 @@
+package cats.derived
+
+import cats.*
+import cats.laws.discipline.{SemigroupKTests, SerializableTests}
+import org.scalacheck.Arbitrary
+
+class SemigroupKSuite extends KittensSuite {
+  import SemigroupKSuite.*
+  import TestDefns.*
+
+  def testSemigroupK(context: String)(implicit
+      complexProduct: SemigroupK[ComplexProduct],
+      caseClassWOption: SemigroupK[CaseClassWOption],
+      boxMul: SemigroupK[BoxMul]
+  ): Unit = {
+    checkAll(s"$context.SemigroupK[ComplexProduct]", SemigroupKTests[ComplexProduct].semigroupK[Char])
+    checkAll(s"$context.SemigroupK[CaseClassWOption]", SemigroupKTests[CaseClassWOption].semigroupK[Char])
+    checkAll(s"$context.SemigroupK[BoxMul]", SemigroupKTests[BoxMul].semigroupK[Char])
+    checkAll(s"$context.SemigroupK is Serializable", SerializableTests.serializable(SemigroupK[ComplexProduct]))
+
+    test(s"$context.SemigroupK respects existing instances") {
+      assert(boxMul.combineK(Box(Mul[Char](5)), Box(Mul[Char](5))) == Box(Mul[Char](25)))
+    }
+  }
+
+  locally {
+    import auto.semigroupK.given
+    summon[SemigroupK[[a] =>> Eval[Option[a]]]]
```

Comment:
Huh why not add a test for it?

Reply:
Woops! Didn't mean to commit that 🤦

Reply:
This was me trying to get the recursive stuff to break. Will remove